Enhance tests

jschnasse opened the pull request:
Tests will fail. But i'd like to share this. The result is not so bad. I'm on to other things now.

jschnasse committed:
Remove unused code, add javadoc

jschnasse committed:
add some assertions and do smaller corrections on testfile
- tests will fail - we have to elaborate

jschnasse committed:
fix travis

jschnasse commented:
View current output in playground. Maybe it would be better to perform tests on that basis.
@acka47 How does it look for you?
[Playground](http://json-ld.org/playground/#startTab=tab-normalized&json-ld=%7B%22%40context%22%3A%7B%22%40context%22%3A%7B%22type%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23type%22%2C%22label%22%3A%22Typ%22%7D%2C%22preferredNameForThePlaceOrGeographicName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForThePlaceOrGeographicName%22%2C%22label%22%3A%22%22%7D%2C%22T1008%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fiflastandards.info%2Fns%2Fisbd%2Fterms%2Fmediatype%2FT1008%22%2C%22label%22%3A%22Kombination%22%7D%2C%22contributorName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2Fcontributor%22%2C%22label%22%3A%22Mitwirkende%22%7D%2C%22lccn%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Flccn%22%2C%22label%22%3A%22Lccn%22%7D%2C%22deu%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Fiso639-2%2Fdeu%22%2C%22label%22%3A%22deutsch%22%7D%2C%22exemplar%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fvocab%2Ffrbr%2Fcore%23exemplar%22%2C%22label%22%3A%22Exemplar%22%7D%2C%22preferredName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredName%22%2C%22label%22%3A%22Name%22%7D%2C%22issued%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fissued%22%2C%22label%22%3A%22Erschienen%22%7D%2C%22hbzId%22%3A%7B%22icon%22%3A%22fa%20fa-bars%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Flobid%2Flv%23hbzID%22%2C%22label%22%3A%22Katalog%20Id%22%7D%2C%22hasPart%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FhasPart%22%2C%22label%22%3A%22Bestandteile%22%7D%2C%22accessScheme%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23accessScheme%22%2C%22label%22%3A%22Sichtbarkeit%20Daten%22%7D%2C%22eissn%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Feissn%22%2C%22label%22%3A%22Eissn%22%7D%2C%22isPartOf%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23isPartOf%22%2C%22label%22%3A%22Bestandteil%20von%22%7D%2C%22P1053%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fiflastandards.info%2Fns%2Fisbd%2Felements%2FP1053%22%2C%22label%22%3A%22Umfang%22%7D%2C%22size%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fdownlode.org%2FCode%2FRDF%2FFile_Properties%2Fschema%23size%22%2C%22label%22%3A%22Gr%C3%B6%C3%9Fe%22%7D%2C%22issn%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Fissn%22%2C%22label%22%3A%22Issn%22%7D%2C%22isbn13%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Fisbn13%22%2C%22label%22%3A%22Isbn13%22%7D%2C%22preferredNameForThePerson%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForThePerson%22%2C%22label%22%3A%22Name%22%7D%2C%22isPartOfName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2FisPartOf%22%2C%22label%22%3A%22Bestandteil%20von%22%7D%2C%22bibliographicCitation%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FbibliographicCitation%22%2C%22label%22%3A%22Ver%C3%B6ffentlichungs%20Details%22%7D%2C%22doi%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Fdoi%22%2C%22label%22%3A%22Doi%22%7D%2C%22objectTimestamp%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23objectTimestamp%22%2C%22label%22%3A%22Objekt%20bearb.%22%7D%2C%22isLike%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.umbel.org%2Fspecifications%2Fvocabulary%23isLike%22%2C%22label%22%3A%22%C3%A4hnlich%20zu%22%7D%2C%22dateOfDeath%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23dateOfDeath%22%2C%22label%22%3A%22Gestorben%20am%22%7D%2C%22translator%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Ftranslator%22%2C%22label%22%3A%22%C3%9Cbersetzer%22%7D%2C%22creatorName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2Fcreator%22%2C%22label%22%3A%22Titelkopie%22%7D%2C%22fulltextOnline%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Flobid.org%2Fvocab%2Flobid%23fulltextOnline%22%2C%22label%22%3A%22Download%22%7D%2C%22importedFrom%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23importedFrom%22%2C%22label%22%3A%22Import%22%7D%2C%22wikipedia%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fmo%2Fwikipedia%22%2C%22label%22%3A%22Wikipedia%22%7D%2C%22preferredNameEntityForThePerson%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameEntityForThePerson%22%2C%22label%22%3A%22%22%7D%2C%22preferredNameForTheConferenceOrEvent%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForTheConferenceOrEvent%22%2C%22label%22%3A%22%22%7D%2C%22parallelEdition%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23parallelEdition%22%2C%22label%22%3A%22Titelkopie%22%7D%2C%22editor%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Feditor%22%2C%22label%22%3A%22Herausgeber%22%7D%2C%22creator%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fcreator%22%2C%22label%22%3A%22Autor%22%7D%2C%22dateOfBirth%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23dateOfBirth%22%2C%22label%22%3A%22Geboren%20am%22%7D%2C%22otherTitleInformation%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Frdvocab.info%2FElements%2FotherTitleInformation%22%2C%22label%22%3A%22Titelzusatz%20weitere%22%7D%2C%22describes%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.openarchives.org%2Fore%2Fterms%2Fdescribes%22%2C%22label%22%3A%22beschreibt%22%7D%2C%22urn%22%3A%7B%22icon%22%3A%22fa%20fa-exchange%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Flobid%2Flv%23urn%22%2C%22label%22%3A%22Urn%22%7D%2C%22hasURN%22%3A%7B%22icon%22%3A%22fa%20fa-exchange%22%2C%22%40id%22%3A%22http%3A%2F%2Fgeni-orca.renci.org%2Fowl%2Ftopology.owl%23hasURN%22%2C%22label%22%3A%22Urn%22%7D%2C%22placeOfPublication%22%3A%7B%22icon%22%3A%22fa%20fa-location-arrow%22%2C%22%40id%22%3A%22http%3A%2F%2Frdvocab.info%2FElements%2FplaceOfPublication%22%2C%22label%22%3A%22Ort%22%7D%2C%22P60489%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Frdaregistry.info%2FElements%2Fu%2FP60489%22%2C%22label%22%3A%22Hochschulschriftenvermerk%22%7D%2C%22describedby%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2007%2F05%2Fpowder-s%23describedby%22%2C%22label%22%3A%22Beschrieben%20durch%22%7D%2C%22ismn%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fmo%2Fismn%22%2C%22label%22%3A%22Ismn%22%7D%2C%22inDataset%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23inDataset%22%2C%22label%22%3A%22%22%7D%2C%22publishScheme%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23publishScheme%22%2C%22label%22%3A%22Sichtbarkeit%20Metadaten%22%7D%2C%22subject%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22icon%22%3A%22fa%20fa-tag%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fsubject%22%2C%22label%22%3A%22Schlagwort%22%7D%2C%22language%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Flanguage%22%2C%22label%22%3A%22Sprache%22%7D%2C%22shortTitle%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2FshortTitle%22%2C%22label%22%3A%22Kurztitel%22%7D%2C%22subjectChain%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Flobid%2Flv%23subjectChain%22%2C%22label%22%3A%22Schlagwortkette%22%7D%2C%22isAggregatedBy%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.openarchives.org%2Fore%2Fterms%2FisAggregatedBy%22%2C%22label%22%3A%22Teil%20von%22%7D%2C%22institution%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fdbpedia.org%2Fontology%2Finstitution%22%2C%22label%22%3A%22Institution%22%7D%2C%22act%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Frelators%2Fact%22%2C%22label%22%3A%22Schauspieler%22%7D%2C%22preferredNameForTheSubjectHeading%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForTheSubjectHeading%22%2C%22label%22%3A%22%22%7D%2C%22modified%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fmodified%22%2C%22label%22%3A%22Zuletzt%20bearbeitet%22%7D%2C%22isPrimaryTopicOf%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fxmlns.com%2Ffoaf%2F0.1%2FisPrimaryTopicOf%22%2C%22label%22%3A%22Gegenstand%20von%22%7D%2C%22contentType%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23contentType%22%2C%22label%22%3A%22Repository%20Typ%22%7D%2C%22publicationStatement%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Frdvocab.info%2FElements%2FpublicationStatement%22%2C%22label%22%3A%22Ort%20und%20Jahr%22%7D%2C%22created%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fcreated%22%2C%22label%22%3A%22Erstellt%20am%22%7D%2C%22format%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fformat%22%2C%22label%22%3A%22Format%22%7D%2C%22P1016%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fiflastandards.info%2Fns%2Fisbd%2Felements%2FP1016%22%2C%22label%22%3A%22Ort%22%7D%2C%22statementOfResponsibility%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Frdvocab.info%2FElements%2FstatementOfResponsibility%22%2C%22label%22%3A%22Verantwortlich%22%7D%2C%22similarTo%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.openarchives.org%2Fore%2Fterms%2FsimilarTo%22%2C%22label%22%3A%22Vgl.%22%7D%2C%22aui%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Frelators%2Faui%22%2C%22label%22%3A%22Autor%20der%20Einleitung%22%7D%2C%22primaryTopic%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fxmlns.com%2Ffoaf%2F0.1%2FprimaryTopic%22%2C%22label%22%3A%22Vgl.%22%7D%2C%22name%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23name%22%2C%22label%22%3A%22Interner%20Name%22%7D%2C%22publisher%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2Fpublisher%22%2C%22label%22%3A%22Herausgeber%22%7D%2C%22tableOfContents%22%3A%7B%22icon%22%3A%22fa%20fa-list%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FtableOfContents%22%2C%22label%22%3A%22Inhaltsverzeichnis%22%7D%2C%22aus%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Frelators%2Faus%22%2C%22label%22%3A%22Screenwriter%22%7D%2C%22oclcnum%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Foclcnum%22%2C%22label%22%3A%22Oclc%22%7D%2C%22preferredNameForTheCorporateBody%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForTheCorporateBody%22%2C%22label%22%3A%22%22%7D%2C%22hasVersion%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FhasVersion%22%2C%22label%22%3A%22Vgl.%22%7D%2C%22isbn%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Fisbn%22%2C%22label%22%3A%22Isbn%22%7D%2C%22generator%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fdownlode.org%2FCode%2FRDF%2FFile_Properties%2Fschema%23generator%22%2C%22label%22%3A%22%22%7D%2C%22P1006%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fiflastandards.info%2Fns%2Fisbd%2Felements%2FP1006%22%2C%22label%22%3A%22Titelzusatz%22%7D%2C%22medium%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fmedium%22%2C%22label%22%3A%22Tr%C3%A4ger%22%7D%2C%22title%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Ftitle%22%2C%22label%22%3A%22Titel%22%7D%2C%22P1004%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fiflastandards.info%2Fns%2Fisbd%2Felements%2FP1004%22%2C%22label%22%3A%22Gesamttitel%22%7D%2C%22seeAlso%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23seeAlso%22%2C%22label%22%3A%22Weitere%20Informationen%22%7D%2C%22itemID%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.openarchives.org%2FOAI%2F2.0%2FitemID%22%2C%22label%22%3A%22Oai%20Id%22%7D%2C%22contributor%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fcontributor%22%2C%22label%22%3A%22Mitwirkende%22%7D%2C%22festschrift%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22icon%22%3A%22octicon%20octicon-gift%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Flobid%2Flv%23Festschrift%22%2C%22label%22%3A%22Festschrift%22%7D%2C%22checksum%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22icon%22%3A%22fa%20fa-barcode%22%2C%22%40id%22%3A%22http%3A%2F%2Fdownlode.org%2FCode%2FRDF%2FFile_Properties%2Fschema%23checksum%22%2C%22label%22%3A%22Pr%C3%BCfsumme%22%7D%2C%22hasTransformer%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23hasTransformer%22%2C%22label%22%3A%22Transformer%22%7D%2C%22legacyId%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23legacyId%22%2C%22label%22%3A%22Id%20Altsystem%22%7D%2C%22subjectName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2Fsubject%22%2C%22label%22%3A%22Schlagwort%22%7D%2C%22preferredNameForTheWork%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForTheWork%22%2C%22label%22%3A%22%22%7D%2C%22hasData%22%3A%7B%22icon%22%3A%22fa%20fa-download%22%2C%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23hasData%22%2C%22label%22%3A%22Download%22%7D%2C%22lastModifiedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23lastModifiedBy%22%2C%22label%22%3A%22Bearbeitet%20von%22%7D%2C%22parentPid%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FisPartOf%22%2C%22label%22%3A%22Bestandteil%20von%22%7D%2C%22prefLabel%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2004%2F02%2Fskos%2Fcore%23prefLabel%22%2C%22label%22%3A%22Label%22%7D%2C%22alternative%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Falternative%22%2C%22label%22%3A%22Titelzusatz%22%7D%2C%22label%22%3A%7B%22icon%22%3A%22fa%20fa-tag%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23label%22%2C%22label%22%3A%22Etikett%22%7D%2C%22aft%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Frelators%2Faft%22%2C%22label%22%3A%22Autor%20des%20Nachwortes%22%7D%2C%22createdBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23createdBy%22%2C%22label%22%3A%22Erstellt%20von%22%7D%2C%22isDescribedBy%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.openarchives.org%2Fore%2Fterms%2FisDescribedBy%22%2C%22label%22%3A%22Beschrieben%20durch%22%7D%2C%22preferredNameForTheFamily%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForTheFamily%22%2C%22label%22%3A%22%22%7D%2C%22sameAs%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2002%2F07%2Fowl%23sameAs%22%2C%22label%22%3A%22Identisch%20zu%22%7D%7D%7D%2C%22%40id%22%3A%22http%3A%2F%2Flobid.org%2Fresource%2FHT018454638%22%2C%22contributor%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F1060772442%22%2C%22preferredName%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%2C%22preferredNameForTheCorporateBody%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%2C%22preferredNameForThePerson%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F172845785%22%2C%22preferredName%22%3A%22Clemens-Schierbaum%2C%20Ursula%2C%201963-%22%2C%22preferredNameForThePerson%22%3A%22Clemens-Schierbaum%2C%20Ursula%2C%201963-%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F2175350-7%22%2C%22preferredName%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%2C%22preferredNameForTheCorporateBody%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%2C%22preferredNameForThePerson%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%7D%5D%2C%22contributorOrder%22%3A%5B%22http%3A%2F%2Fd-nb.info%2Fgnd%2F172845785%22%2C%22http%3A%2F%2Fd-nb.info%2Fgnd%2F1060772442%22%2C%22http%3A%2F%2Fd-nb.info%2Fgnd%2F2175350-7%22%5D%2C%22describedby%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Flobid.org%2Fresource%2FHT018454638%2Fabout%22%2C%22created%22%3A%2220141111%22%2C%22modified%22%3A%2220150303%22%7D%5D%2C%22exemplar%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Flobid.org%2Fitem%2FHT018454638%3ADE-5-39%3AGb%25207470%252F201%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Flobid.org%2Fitem%2FHT018454638%3ADE-Kn28%3AFc%25206601%22%7D%5D%2C%22language%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Fiso639-2%2Fdeu%22%7D%5D%2C%22medium%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Frdvocab.info%2FtermList%2FRDAproductionMethod%2F1010%22%7D%5D%2C%22subject%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F1060772442%22%2C%22preferredName%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%2C%22preferredNameForTheCorporateBody%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%2C%22preferredNameForThePerson%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F2175350-7%22%2C%22preferredName%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%2C%22preferredNameForTheCorporateBody%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%2C%22preferredNameForThePerson%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%7D%5D%2C%22subjectOrder%22%3A%5B%22http%3A%2F%2Fd-nb.info%2Fgnd%2F1060772442%22%2C%22http%3A%2F%2Fd-nb.info%2Fgnd%2F2175350-7%22%2C%22Geschichte%22%5D%2C%22type%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FBibliographicResource%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2FBook%22%7D%5D%7D&context=%7B%7D)

acka47 commented:
Already looks quite good. Some things have to be tweaked, though:
- There are two `@context` objects.
- `@id` keys are shown in the data for keys where the context already defines values as @id's (@type, medium, language etc.). Example: 

``` javascript
"medium": [
    {
      "@id": "http://rdvocab.info/termList/RDAproductionMethod/1010"
    }
  ]
```

I'd expect:

``` javascript
"medium": [ "http://rdvocab.info/termList/RDAproductionMethod/1010" ]
```

acka47 commented:
I don't know what you mean with "performing tests on the basis of JSON-LD playground". However, for me the playground doesn't improve my testing experience as I just look at the output (whihc is the playground's input in this case). I don't know whether it might come handy in the future.

jschnasse commented:
Ok! I was thinking about a test, where JSON is converted back to ntriple and compared to the original input.

jschnasse commented:
> ' @id keys are shown in the data for keys where the context already defines values as @id's '

Ah, I didn't know that.
How about my idea to unify the serialization of all fields, so that for type @id always can be expected the following form

```
[
    {
        @id:....
        prefLabel:....
        .......
    }, ....
]
```

So, you always have arrays. Each @id comes with a human-readable label that is also used to boost search results.

For all other types you have 

```
[
   "str1", "str2"
]
```

acka47 commented:
> Ok! I was thinking about a test, where JSON is converted back to ntriple and compared to the original input.

Makes sense for me.

acka47 commented:
> How about my idea to unify the serialization of all fields, so that for type @id always can be expected ...
> So, you always have arrays. Each @id comes with a human-readable label that is also used to boost search results.

This also makes sense for me but I will have to think about it and discuss with @fsteeg and @dr0i .

jschnasse committed:
remove dublicated @context

jschnasse committed:
edit testcase to meet current configuration
- fieldnames can be configured by passing the correct
context.json and labels.json to the converter. For the moment
the converter uses a standard similar to https://frl.publisso.de/tools/etikett

jschnasse commented:
Ntriple to json conversion will produce a result similar to https://github.com/hbz/lobid-rdf-to-json/blob/master/src/test/resources/hbz01.es.json (see modification history)

[Playground](http://json-ld.org/playground/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%7B%22type%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23type%22%2C%22label%22%3A%22Typ%22%7D%2C%22preferredNameForThePlaceOrGeographicName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForThePlaceOrGeographicName%22%2C%22label%22%3A%22%22%7D%2C%22T1008%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fiflastandards.info%2Fns%2Fisbd%2Fterms%2Fmediatype%2FT1008%22%2C%22label%22%3A%22Kombination%22%7D%2C%22contributorName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2Fcontributor%22%2C%22label%22%3A%22Mitwirkende%22%7D%2C%22lccn%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Flccn%22%2C%22label%22%3A%22Lccn%22%7D%2C%22deu%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Fiso639-2%2Fdeu%22%2C%22label%22%3A%22deutsch%22%7D%2C%22exemplar%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fvocab%2Ffrbr%2Fcore%23exemplar%22%2C%22label%22%3A%22Exemplar%22%7D%2C%22preferredName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredName%22%2C%22label%22%3A%22Name%22%7D%2C%22issued%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fissued%22%2C%22label%22%3A%22Erschienen%22%7D%2C%22hbzId%22%3A%7B%22icon%22%3A%22fa%20fa-bars%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Flobid%2Flv%23hbzID%22%2C%22label%22%3A%22Katalog%20Id%22%7D%2C%22hasPart%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FhasPart%22%2C%22label%22%3A%22Bestandteile%22%7D%2C%22accessScheme%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23accessScheme%22%2C%22label%22%3A%22Sichtbarkeit%20Daten%22%7D%2C%22eissn%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Feissn%22%2C%22label%22%3A%22Eissn%22%7D%2C%22isPartOf%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23isPartOf%22%2C%22label%22%3A%22Bestandteil%20von%22%7D%2C%22P1053%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fiflastandards.info%2Fns%2Fisbd%2Felements%2FP1053%22%2C%22label%22%3A%22Umfang%22%7D%2C%22size%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fdownlode.org%2FCode%2FRDF%2FFile_Properties%2Fschema%23size%22%2C%22label%22%3A%22Gr%C3%B6%C3%9Fe%22%7D%2C%22issn%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Fissn%22%2C%22label%22%3A%22Issn%22%7D%2C%22isbn13%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Fisbn13%22%2C%22label%22%3A%22Isbn13%22%7D%2C%22preferredNameForThePerson%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForThePerson%22%2C%22label%22%3A%22Name%22%7D%2C%22isPartOfName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2FisPartOf%22%2C%22label%22%3A%22Bestandteil%20von%22%7D%2C%22bibliographicCitation%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FbibliographicCitation%22%2C%22label%22%3A%22Ver%C3%B6ffentlichungs%20Details%22%7D%2C%22doi%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Fdoi%22%2C%22label%22%3A%22Doi%22%7D%2C%22objectTimestamp%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23objectTimestamp%22%2C%22label%22%3A%22Objekt%20bearb.%22%7D%2C%22isLike%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.umbel.org%2Fspecifications%2Fvocabulary%23isLike%22%2C%22label%22%3A%22%C3%A4hnlich%20zu%22%7D%2C%22dateOfDeath%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23dateOfDeath%22%2C%22label%22%3A%22Gestorben%20am%22%7D%2C%22translator%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Ftranslator%22%2C%22label%22%3A%22%C3%9Cbersetzer%22%7D%2C%22creatorName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2Fcreator%22%2C%22label%22%3A%22Titelkopie%22%7D%2C%22fulltextOnline%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Flobid.org%2Fvocab%2Flobid%23fulltextOnline%22%2C%22label%22%3A%22Download%22%7D%2C%22importedFrom%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23importedFrom%22%2C%22label%22%3A%22Import%22%7D%2C%22wikipedia%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fmo%2Fwikipedia%22%2C%22label%22%3A%22Wikipedia%22%7D%2C%22preferredNameEntityForThePerson%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameEntityForThePerson%22%2C%22label%22%3A%22%22%7D%2C%22preferredNameForTheConferenceOrEvent%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForTheConferenceOrEvent%22%2C%22label%22%3A%22%22%7D%2C%22parallelEdition%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23parallelEdition%22%2C%22label%22%3A%22Titelkopie%22%7D%2C%22editor%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Feditor%22%2C%22label%22%3A%22Herausgeber%22%7D%2C%22creator%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fcreator%22%2C%22label%22%3A%22Autor%22%7D%2C%22dateOfBirth%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23dateOfBirth%22%2C%22label%22%3A%22Geboren%20am%22%7D%2C%22otherTitleInformation%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Frdvocab.info%2FElements%2FotherTitleInformation%22%2C%22label%22%3A%22Titelzusatz%20weitere%22%7D%2C%22describes%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.openarchives.org%2Fore%2Fterms%2Fdescribes%22%2C%22label%22%3A%22beschreibt%22%7D%2C%22urn%22%3A%7B%22icon%22%3A%22fa%20fa-exchange%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Flobid%2Flv%23urn%22%2C%22label%22%3A%22Urn%22%7D%2C%22hasURN%22%3A%7B%22icon%22%3A%22fa%20fa-exchange%22%2C%22%40id%22%3A%22http%3A%2F%2Fgeni-orca.renci.org%2Fowl%2Ftopology.owl%23hasURN%22%2C%22label%22%3A%22Urn%22%7D%2C%22placeOfPublication%22%3A%7B%22icon%22%3A%22fa%20fa-location-arrow%22%2C%22%40id%22%3A%22http%3A%2F%2Frdvocab.info%2FElements%2FplaceOfPublication%22%2C%22label%22%3A%22Ort%22%7D%2C%22P60489%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Frdaregistry.info%2FElements%2Fu%2FP60489%22%2C%22label%22%3A%22Hochschulschriftenvermerk%22%7D%2C%22describedby%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2007%2F05%2Fpowder-s%23describedby%22%2C%22label%22%3A%22Beschrieben%20durch%22%7D%2C%22ismn%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fmo%2Fismn%22%2C%22label%22%3A%22Ismn%22%7D%2C%22inDataset%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23inDataset%22%2C%22label%22%3A%22%22%7D%2C%22publishScheme%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23publishScheme%22%2C%22label%22%3A%22Sichtbarkeit%20Metadaten%22%7D%2C%22subject%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22icon%22%3A%22fa%20fa-tag%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fsubject%22%2C%22label%22%3A%22Schlagwort%22%7D%2C%22language%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Flanguage%22%2C%22label%22%3A%22Sprache%22%7D%2C%22shortTitle%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2FshortTitle%22%2C%22label%22%3A%22Kurztitel%22%7D%2C%22subjectChain%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Flobid%2Flv%23subjectChain%22%2C%22label%22%3A%22Schlagwortkette%22%7D%2C%22isAggregatedBy%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.openarchives.org%2Fore%2Fterms%2FisAggregatedBy%22%2C%22label%22%3A%22Teil%20von%22%7D%2C%22institution%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fdbpedia.org%2Fontology%2Finstitution%22%2C%22label%22%3A%22Institution%22%7D%2C%22act%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Frelators%2Fact%22%2C%22label%22%3A%22Schauspieler%22%7D%2C%22preferredNameForTheSubjectHeading%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForTheSubjectHeading%22%2C%22label%22%3A%22%22%7D%2C%22modified%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fmodified%22%2C%22label%22%3A%22Zuletzt%20bearbeitet%22%7D%2C%22isPrimaryTopicOf%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fxmlns.com%2Ffoaf%2F0.1%2FisPrimaryTopicOf%22%2C%22label%22%3A%22Gegenstand%20von%22%7D%2C%22contentType%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23contentType%22%2C%22label%22%3A%22Repository%20Typ%22%7D%2C%22publicationStatement%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Frdvocab.info%2FElements%2FpublicationStatement%22%2C%22label%22%3A%22Ort%20und%20Jahr%22%7D%2C%22created%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fcreated%22%2C%22label%22%3A%22Erstellt%20am%22%7D%2C%22format%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fformat%22%2C%22label%22%3A%22Format%22%7D%2C%22P1016%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fiflastandards.info%2Fns%2Fisbd%2Felements%2FP1016%22%2C%22label%22%3A%22Ort%22%7D%2C%22statementOfResponsibility%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Frdvocab.info%2FElements%2FstatementOfResponsibility%22%2C%22label%22%3A%22Verantwortlich%22%7D%2C%22similarTo%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.openarchives.org%2Fore%2Fterms%2FsimilarTo%22%2C%22label%22%3A%22Vgl.%22%7D%2C%22aui%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Frelators%2Faui%22%2C%22label%22%3A%22Autor%20der%20Einleitung%22%7D%2C%22primaryTopic%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fxmlns.com%2Ffoaf%2F0.1%2FprimaryTopic%22%2C%22label%22%3A%22Vgl.%22%7D%2C%22name%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23name%22%2C%22label%22%3A%22Interner%20Name%22%7D%2C%22publisher%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2Fpublisher%22%2C%22label%22%3A%22Herausgeber%22%7D%2C%22tableOfContents%22%3A%7B%22icon%22%3A%22fa%20fa-list%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FtableOfContents%22%2C%22label%22%3A%22Inhaltsverzeichnis%22%7D%2C%22aus%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Frelators%2Faus%22%2C%22label%22%3A%22Screenwriter%22%7D%2C%22oclcnum%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Foclcnum%22%2C%22label%22%3A%22Oclc%22%7D%2C%22preferredNameForTheCorporateBody%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForTheCorporateBody%22%2C%22label%22%3A%22%22%7D%2C%22hasVersion%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FhasVersion%22%2C%22label%22%3A%22Vgl.%22%7D%2C%22isbn%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2Fisbn%22%2C%22label%22%3A%22Isbn%22%7D%2C%22generator%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fdownlode.org%2FCode%2FRDF%2FFile_Properties%2Fschema%23generator%22%2C%22label%22%3A%22%22%7D%2C%22P1006%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fiflastandards.info%2Fns%2Fisbd%2Felements%2FP1006%22%2C%22label%22%3A%22Titelzusatz%22%7D%2C%22medium%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fmedium%22%2C%22label%22%3A%22Tr%C3%A4ger%22%7D%2C%22title%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Ftitle%22%2C%22label%22%3A%22Titel%22%7D%2C%22P1004%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fiflastandards.info%2Fns%2Fisbd%2Felements%2FP1004%22%2C%22label%22%3A%22Gesamttitel%22%7D%2C%22seeAlso%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23seeAlso%22%2C%22label%22%3A%22Weitere%20Informationen%22%7D%2C%22itemID%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.openarchives.org%2FOAI%2F2.0%2FitemID%22%2C%22label%22%3A%22Oai%20Id%22%7D%2C%22contributor%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Fcontributor%22%2C%22label%22%3A%22Mitwirkende%22%7D%2C%22festschrift%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22icon%22%3A%22octicon%20octicon-gift%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Flobid%2Flv%23Festschrift%22%2C%22label%22%3A%22Festschrift%22%7D%2C%22checksum%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22icon%22%3A%22fa%20fa-barcode%22%2C%22%40id%22%3A%22http%3A%2F%2Fdownlode.org%2FCode%2FRDF%2FFile_Properties%2Fschema%23checksum%22%2C%22label%22%3A%22Pr%C3%BCfsumme%22%7D%2C%22hasTransformer%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23hasTransformer%22%2C%22label%22%3A%22Transformer%22%7D%2C%22legacyId%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23legacyId%22%2C%22label%22%3A%22Id%20Altsystem%22%7D%2C%22subjectName%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2Fsubject%22%2C%22label%22%3A%22Schlagwort%22%7D%2C%22preferredNameForTheWork%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForTheWork%22%2C%22label%22%3A%22%22%7D%2C%22hasData%22%3A%7B%22icon%22%3A%22fa%20fa-download%22%2C%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23hasData%22%2C%22label%22%3A%22Download%22%7D%2C%22lastModifiedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23lastModifiedBy%22%2C%22label%22%3A%22Bearbeitet%20von%22%7D%2C%22parentPid%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FisPartOf%22%2C%22label%22%3A%22Bestandteil%20von%22%7D%2C%22prefLabel%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2004%2F02%2Fskos%2Fcore%23prefLabel%22%2C%22label%22%3A%22Label%22%7D%2C%22alternative%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2Falternative%22%2C%22label%22%3A%22Titelzusatz%22%7D%2C%22label%22%3A%7B%22icon%22%3A%22fa%20fa-tag%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23label%22%2C%22label%22%3A%22Etikett%22%7D%2C%22aft%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Frelators%2Faft%22%2C%22label%22%3A%22Autor%20des%20Nachwortes%22%7D%2C%22createdBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fhbz-nrw.de%2Fregal%23createdBy%22%2C%22label%22%3A%22Erstellt%20von%22%7D%2C%22isDescribedBy%22%3A%7B%22%40type%22%3A%22%40id%22%2C%22%40id%22%3A%22http%3A%2F%2Fwww.openarchives.org%2Fore%2Fterms%2FisDescribedBy%22%2C%22label%22%3A%22Beschrieben%20durch%22%7D%2C%22preferredNameForTheFamily%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fstandards%2Felementset%2Fgnd%23preferredNameForTheFamily%22%2C%22label%22%3A%22%22%7D%2C%22sameAs%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F2002%2F07%2Fowl%23sameAs%22%2C%22label%22%3A%22Identisch%20zu%22%7D%7D%2C%22%40id%22%3A%22http%3A%2F%2Flobid.org%2Fresource%2FHT018454638%22%2C%22contributor%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F1060772442%22%2C%22preferredName%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%2C%22preferredNameForTheCorporateBody%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%2C%22preferredNameForThePerson%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F172845785%22%2C%22preferredName%22%3A%22Clemens-Schierbaum%2C%20Ursula%2C%201963-%22%2C%22preferredNameForThePerson%22%3A%22Clemens-Schierbaum%2C%20Ursula%2C%201963-%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F2175350-7%22%2C%22preferredName%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%2C%22preferredNameForTheCorporateBody%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%2C%22preferredNameForThePerson%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%7D%5D%2C%22contributorOrder%22%3A%5B%22http%3A%2F%2Fd-nb.info%2Fgnd%2F172845785%22%2C%22http%3A%2F%2Fd-nb.info%2Fgnd%2F1060772442%22%2C%22http%3A%2F%2Fd-nb.info%2Fgnd%2F2175350-7%22%5D%2C%22describedby%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Flobid.org%2Fresource%2FHT018454638%2Fabout%22%2C%22created%22%3A%2220141111%22%2C%22modified%22%3A%2220150303%22%7D%5D%2C%22exemplar%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Flobid.org%2Fitem%2FHT018454638%3ADE-5-39%3AGb%25207470%252F201%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Flobid.org%2Fitem%2FHT018454638%3ADE-Kn28%3AFc%25206601%22%7D%5D%2C%22language%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Fid.loc.gov%2Fvocabulary%2Fiso639-2%2Fdeu%22%7D%5D%2C%22medium%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Frdvocab.info%2FtermList%2FRDAproductionMethod%2F1010%22%7D%5D%2C%22subject%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F1060772442%22%2C%22preferredName%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%2C%22preferredNameForTheCorporateBody%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%2C%22preferredNameForThePerson%22%3A%22Pfarrei%20St.%20Joseph%20und%20St.%20Antonius%20Bergisch%20Gladbach%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Fd-nb.info%2Fgnd%2F2175350-7%22%2C%22preferredName%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%2C%22preferredNameForTheCorporateBody%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%2C%22preferredNameForThePerson%22%3A%22Pfarrei%20Sankt%20Antonius%20Abbas%20Herkenrath%22%7D%5D%2C%22subjectOrder%22%3A%5B%22http%3A%2F%2Fd-nb.info%2Fgnd%2F1060772442%22%2C%22http%3A%2F%2Fd-nb.info%2Fgnd%2F2175350-7%22%2C%22Geschichte%22%5D%2C%22type%22%3A%5B%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2FBibliographicResource%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Fpurl.org%2Fontology%2Fbibo%2FBook%22%7D%5D%7D)

jschnasse committed:
Merge branch 'master' of https://github.com/hbz/lobid-rdf-to-json
